### PR TITLE
New Python Reference App.

### PR DESCRIPTION
**Pull request by Affinidi-Grajesh**
Added New Python Reference app with Authlib 
1. Flask
2. FastAPI
3. Starlette

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,21 +5,30 @@
...
+    "install:affinidi-flask-authlib": "pip install -r samples/affinidi-flask-authlib/requirements.txt",
+    "install:affinidi-fastapi-authlib": "pip install -r samples/affinidi-fastapi-authlib/requirements.txt",
+    "install:affinidi-starlette-authlib": "pip install -r samples/affinidi-starlette-authlib/requirements.txt",
     "start": "concurrently -n w: \"npm:start:*(!template)\"",
     "start:template": "npm run --prefix generator/template dev -- -p 3000",
     "start:affinidi-nextjs-nextauthjs": "npm run --prefix samples/affinidi-nextjs-nextauthjs dev -- -p 3001",
     "start:auth0-nextjs-nextauthjs": "npm run --prefix samples/auth0-nextjs-nextauthjs dev -- -p 3002",
     "start:affinidi-django-authlib": "python3 samples/affinidi-django-authlib/manage.py migrate && python3 samples/affinidi-django-authlib/manage.py runserver 8000",
     "start:auth0-django-authlib": "python3 samples/auth0-django-authlib/manage.py migrate && python3 samples/auth0-django-authlib/manage.py runserver 8001",
+    "start:affinidi-flask-authlib": "cd samples/affinidi-flask-authlib && python3 -m flask run",
```
Comment (pulkitb2) [inline package.json:29]: Can we provide different ports for all these apps to avoid collision of ports

Comment (Affinidi-Grajesh) [reply package.json]: yes, I have updated ports for all reference application. 
FastAPI-8200
flask-8100
starlette-8300

Also updated Readme

**Review by pulkitb2 — DISMISSED** [review]

**Review by pulkitb2 — APPROVED** [review]

File: generator/fastapi/generate.mjs [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+  const overrides = (await fs.readdir(overridesPath, { withFileTypes: true }))
+    .filter((i) => i.isDirectory())
+    .map((i) => i.name)
+    .sort()
+
+  console.log(`Detected samples: ${overrides.join(', ')}`)
+
+  for (const [i, sample] of overrides.entries()) {
+    console.log(`\nGenerating "${sample}" sample`)
+    const port = 3000 + i + 1
```
Comment (pulkitb2) [inline generator/fastapi/generate.mjs:31]: we can change this port value

File: generator/flask/generate.mjs [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+  const overrides = (await fs.readdir(overridesPath, { withFileTypes: true }))
+    .filter((i) => i.isDirectory())
+    .map((i) => i.name)
+    .sort()
+
+  console.log(`Detected samples: ${overrides.join(', ')}`)
+
+  for (const [i, sample] of overrides.entries()) {
+    console.log(`\nGenerating "${sample}" sample`)
+    const port = 3000 + i + 1
```
Comment (pulkitb2) [inline generator/flask/generate.mjs:31]: Same for here, we can change this port value or remove if it is not being used

File: generator/starlette/generate.mjs [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+  const overrides = (await fs.readdir(overridesPath, { withFileTypes: true }))
+    .filter((i) => i.isDirectory())
+    .map((i) => i.name)
+    .sort()
+
+  console.log(`Detected samples: ${overrides.join(', ')}`)
+
+  for (const [i, sample] of overrides.entries()) {
+    console.log(`\nGenerating "${sample}" sample`)
+    const port = 3000 + i + 1
```
Comment (pulkitb2) [inline generator/starlette/generate.mjs:31]: Same for here, we can change this port value or remove if it is not being used

**Review by pulkitb2 — CHANGES_REQUESTED** [review]

File: generator/starlette/generate.mjs (hunk 4 above)
Comment (Affinidi-Grajesh) [reply generator/starlette/generate.mjs]: done, removed.

File: generator/flask/generate.mjs (hunk 3 above)
Comment (Affinidi-Grajesh) [reply generator/flask/generate.mjs]: done, removed.

**Review by pulkitb2 — APPROVED** [review]

**Pull request merged**